src/main/java/com/fundation/logic/view/PlayerJFrame.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view;
+
+import com.sun.jna.NativeLibrary;
+import uk.co.caprica.vlcj.component.EmbeddedMediaPlayerComponent;
+import uk.co.caprica.vlcj.player.MediaPlayer;
+import uk.co.caprica.vlcj.player.MediaPlayerEventAdapter;
+import uk.co.caprica.vlcj.runtime.RuntimeUtil;
+import javax.swing.*;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ChangeListener;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.File;
+
+/**
+ * Implements the player frame with playing control buttons
+ *
+ * @author Melissa Román
+ * @version 1.0
+ */
+class PlayerJFrame extends JFrame {
+
+    private JLabel playButton = new JLabel();
+    private JLabel stopButton = new JLabel();
+    private JLabel pauseButton = new JLabel();
+    private int vol = 50;
+
+    private JPanel headerPanel = new JPanel();
+    private JPanel playingPanel = new JPanel();
+    private JPanel bottomPanel = new JPanel();
+
+    private JSlider progressBar = new JSlider(0, 100, 0);
+    private JSlider volumen = new JSlider(0, 100, vol);
+
+    private boolean AccionProgres = true;
+
+    private EmbeddedMediaPlayerComponent player = new EmbeddedMediaPlayerComponent();
+    private File file1 = new File("resources/Wildlife.wmv");
+
+    static {
+        NativeLibrary.addSearchPath(RuntimeUtil.getLibVlcLibraryName(), "C:/Program Files/VideoLAN/VLC/");
```

Comment:
Don't use a generic path.

Reply:
It is necessary to use a relative path because the fileToBePlayed may not exist in the absolute path on other person system